hydrologic-modeling.md

```diff
@@ -0,0 +1,32 @@
+### Who am I?
+
+I am [Joe Hamman](http://joehamman.com/about/) and I am a Project Scientist in the [Computational Hydrology Group](https://ncar.github.io/hydrology/) at the [National Center for Atmospheric Research](https://ncar.ucar.edu/). I am a core developer of the [Xarray](http://xarray.pydata.org) project and a contributing member of the [Pangeo](http://pangeo-data.org/) project. I study subjects in the areas of climate change, hydrology, and water resource engineering.
+
+### What problem am I trying to solve?
+
+Climate change will bring widespread changes to the hydrologic cycle. While previous research has shown what the first order impacts of climate change will look like in managed and natural hydrologic systems, we don't have a very good understanding of the characteristic uncertainties that come from our choice of tools (climate and hydrologic models). We are working on methods to expose and constrain methodological uncertainties in the climate impacts modeling paradigm for water resource applications. This includes developing and analyzing large ensembles of climate projections.
```

Comment:
Are there additional details that you would say to someone within your field, like you analyze coastal erosion models with class-two models or something?